pkg/commands/executor.go

```diff
@@ -54,7 +54,6 @@ func NewExecutor(buildInfo BuildInfo) *Executor {
 	e := &Executor{
 		cfg:       config.NewDefault(),
 		buildInfo: buildInfo,
-		dbManager: lintersdb.NewManager(nil, nil),
```

Comment:
it must help to replace *cfg.Config in linters constructor with specific fields
(and remove my `lintConfigDirGetter`